gen3_tracker/meta/dataframer.py

```diff
@@ -0,0 +1,320 @@
+import json
+import pathlib
+import sqlite3
+from typing import Generator
+
+import inflection
+import ndjson
+import numpy as np
+import pandas as pd
+
+
+class LocalFHIRDatabase:
```

Comment:
Seems like the purpose of this sqlite DB is to get around having to use Gen3Query SDK calls to get summary statistics from guppy. Curious why this decision was made.

If there is a delta between what is in Elastic and what has been brought down locally, issues could arise.

Reply:
Yes,  thats a trade-off.   The purpose is to allow the user to QA metadata by
* validating
* visualizing
* showing dataframe
**before** the data is sent to the server